docs/index.html

```diff
@@ -53,7 +53,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Can be a slow coder.</li>
         <li>Doesn't take themselves too seriously.</li>
         <li>Isn't defined by myopic Tweetstorms by clueless VCs.</li>
-        <li>Fails from time to time.</li>
+        <li>Fails from time to time, and learns from those failures.</li>
```

Comment:
Slight tweak that I think is more in-tune with the approach of this document :)

```suggestion
        <li>Fails from time to time.</li>
        <li>Learns from their failures.</li>
```